rust/connlib/tunnel/src/device_channel/tun_linux.rs

```diff
@@ -208,7 +213,8 @@ async fn set_iface_config(config: InterfaceConfig, handle: Handle) -> Result<()>
 
     handle.link().set(index).up().execute().await?;
 
-    Ok(res_v4.or(res_v6)?)
+    res_v4.or(res_v6)?;
+    Ok(())
```

Comment:
This is silly but it looked kind of weird with a unit type.

Reply:
I always prefer doing all error handling with `?` and have a "simple" return expression at the end.